pwndbg/heap/ptmalloc.py

```diff
@@ -347,42 +347,42 @@ def get_heap_boundaries(self, addr=None):
         Get the boundaries of the heap containing `addr`. Returns the brk region for
         adresses inside it or a fake Page for the containing heap for non-main arenas.
         """
+        page = pwndbg.memory.Page(0, 0, 0, 0)
         brk = self.get_region()
         if addr is None or brk.vaddr < addr < brk.vaddr + brk.memsz:
-            return brk
+            # Occasionally, the [heap] vm region and the actual start of the heap are
```

Comment:
Can we add an explanation for that (or a link to source code that explains it?) e.g. in next PR?